tests/sentry/integrations/jira/models/test_jira_schema.py

```diff
@@ -0,0 +1,13 @@
+from fixtures.integrations.jira.stub_client import StubJiraApiClient
+from sentry.integrations.jira.models.create_issue_metadata import JiraIssueTypeMetadata
+from sentry.testutils.cases import TestCase
+
+
+class TestJiraSchema(TestCase):
```

Comment:
Mentioned this before but to have it on the PR too: we need better testing for these, something that looks like real life config that what stub currently have. As you monitor these can you please update the stub meta data as well?

Reply:
Yes, agreed! I'm trying to think of a way to decompose the schema in a way to make individual parts testable vs just having a massive schema acting as a superset of everything, so I'll circle back around to this shortly.